### ISSUE TITLE
Picture from multiple kinects

### ISSUE THREAD

**Issue by ArkadiyEpanov** [Kinect/PyKinect2#91]
Hello! how can I get a picture from multiple kinects? Couldn't find any information

**Comment by KonstantinosAng** [comment]
It is answered here https://github.com/Kinect/PyKinect2/issues/75.

**Comment by ArkadiyEpanov** [comment]
> It is answered here #75.

Thank you, is there a chance to implement this with python on windows? the example uses linux.
For my purpose, I only need a depth map, with no other functions

**Comment by KonstantinosAng** [comment]
I dont think it is possible in windows with PyKinect2, try to ask a question in stackoverflow to see if someone has figured out a solution.

**Comment by ArkadiyEpanov** [comment]
A small guide how to connect two kinects, python+windows+freenect2:
https://stackoverflow.com/questions/67902941/how-to-call-multiple-kinect2-device-with-pykinect2/67914523#67914523
